### ISSUE TITLE
Remove all shaded dependencies

### ISSUE THREAD

**Issue by quaso** [kagkarlsson/db-scheduler#242]
https://github.com/kagkarlsson/db-scheduler/blob/cb6fd845144ccc1eda6ae37535c60bbf6b020062/pom.xml#L165

This is maven antipattern

**Comment by kagkarlsson** [comment]
Hi! It has been mentioned previously by another user. Is it causing you problems?

**Comment by quaso** [comment]
> Hi! It has been mentioned previously by another user. Is it causing you problems?

Yes, because it is impossible to override cron-utils version.

**Comment by kagkarlsson** [comment]
Yeah, I see your point. What is your reason to override the version of cron-utils?

**Comment by quaso** [comment]
Currently your outdated version of cron-utils has https://nvd.nist.gov/vuln/detail/CVE-2021-41269 vulnerability reported. Quick and dirty fix is to update to latest version. 
The question is, if you going to repeat it in the future? Better option is to remove shaded dependencies completely.

**Comment by kagkarlsson** [comment]
Yeah, I can see that. I recently updated that dependency, so if you update to v10.4 you should get rid of the CVE at least. 
But your point stands for future unknown CVEs.

**Comment by quaso** [comment]
Version 10.4 is still reporting the same vulnerability

db-scheduler-10.4.jar/META-INF/maven/com.cronutils/cron-utils/pom.xml (pkg:maven/com.cronutils/cron-utils@9.1.3, cpe:2.3:a:cron-utils_project:cron-utils:9.1.3:*:*:*:*:*:*:*, cpe:2.3:a:cron_project:cron:9.1.3:*:*:*:*:*:*:*) : CVE-2021-41269

**Comment by kagkarlsson** [comment]
Ah, crap, my mistake, that change did not make it into the latest release

**Comment by kagkarlsson** [comment]
Closing as out-of-scope
